velox/vector/arrow/Bridge.cpp

```diff
@@ -1383,7 +1387,7 @@ void exportToArrow(
   // No additional metadata for now.
   arrowSchema.metadata = nullptr;
 
-  // All supported types are semantically nullable.
+  // All supported types except map are semantically nullable.
```

Comment:
conceptually maps are also nullable - there is nothing wrong in having a map which is NULL.

My understanding is just that what represents a null map is stored in the top level array, not in the "structs" array, since the structs array only exists because of how arrow decided to implement this. 

@whutjs could you update the comment accordingly?

Reply:
I'm not sure I fully understand what you what to change, so I just restore this comment to the original version.